### PR TITLE
Bug fix in PFPUtils.

### PR DESCRIPTION
**Pull request by brballer**
Wanwei found this bug and proposed the solution.

### CODE REVIEW

File: larreco/RecoAlg/TCAlg/PFPUtils.cxx [hunk 1]
```diff
@@ -1891,6 +1891,7 @@ namespace tca {
         auto& tp = slc.tjs[tp3d.TjID - 1].Pts[tp3d.TPIndex];
         sfTPs.push_back(tp);
         wire = std::nearbyint(tp.Pos[0]);
+        if (wire >= slc.nWires[pln]) break;
```
Comment (absolution1) [inline larreco/RecoAlg/TCAlg/PFPUtils.cxx:1894]: Hi @brballer 
Would the geometry service be helpful here or would Thea function calls be too slow?  I think there are functions to find wires for a given position/plane: https://nusoft.fnal.gov/larsoft/doxsvn/html/classgeo_1_1GeometryCore.html#a7f9644be6b7a420fbeeb54c9ded33d62

**Comment by lgarren** [comment]
@absolution1 Dom, are you happy with Bruce's reply?

**Comment by knoepfel** [comment]
This PR should be approved conditional upon L2 approval.

**Comment by absolution1** [comment]
Hi.  Apologies, I was off ill yesterday.  All sounds good to me.

**Comment by absolution1** [comment]
approve

**Comment by lgarren** [comment]
approve

**Pull request merged**